includes/object-cache.php

```diff
@@ -686,18 +687,22 @@ protected function connect_using_phpredis( $parameters ) {
 
             $this->diagnostics[ 'shards' ] = WP_REDIS_SHARDS;
         } elseif ( defined( 'WP_REDIS_CLUSTER' ) ) {
-            $args = [
-                'cluster' => $this->build_cluster_connection_array(),
-                'timeout' => $parameters['timeout'],
-                'read_timeout' => $parameters['read_timeout'],
-                'persistent' => $parameters['persistent'],
-            ];
+            if ( $parameters['cluster_name'] ) {
```

Comment:
I don't want to introduce *another* constant, please use `WP_REDIS_CLUSTER` and check the type.

Reply:
Done